manage

```diff
@@ -113,6 +113,9 @@ usage () {
     $0 run -d acapy -t @SmokeTest -t @P1    - Run the tests tagged @SmokeTest and/or @P1 (priority 1) using all ACA-Py agents
     $0 run -d acapy -b mobile -n -t @MobileTest  - Run the mobile tests using ngrok endpoints
 
+  runset - Run the set of tests for a combination of Test Agents using the parameters in the daily, GitHub Action run "runsets".
+      To see full set of options for the "runset" command, execute "./manage runset -h"
```

Comment:
I get an error running the `-h` option:
```
$ ./manage runset -h
Error: AATH RunSet not found, must be one of:
verity-afj verity-dotnet acapy-afgo acapy-afj acapy-aip10 acapy-aip20 acapy-ariesvcx acapy-dotnet-javascript acapy-dotnet acapy-findy acapy-verity acapy afgo-acapy afgo afj-acapy afj-dotnet afj-findy afj ariesvcx-acapy ariesvcx-ariesvcx dotnet-acapy dotnet-findy dotnet-javascript dotnet findy-acapy findy-dotnet findy-javascript-dotnet findy-javascript findy 
```

Reply:
Dang...I'll take a look.  Thanks!

Reply:
Found it.  Ah, the processing of command line args -- such fun.

Please re-review...